osbs/build/spec.py

```diff
@@ -431,3 +431,154 @@ def validate(self):
 
     def __repr__(self):
         return "Spec(%s)" % self.__dict__
+
+
+class BuildUserParams(object):
+    def __init__(self):
+        self.git_uri = BuildParam('git_uri')
+        self.git_ref = BuildParam('git_ref', default=DEFAULT_GIT_REF)
+        self.git_branch = BuildParam('git_branch')
+        self.user = UserParam()
+        self.use_auth = BuildParam("use_auth", allow_none=True)
```

Comment:
`use_auth` doesn't seem like a user parameter. I'd expect this to be an environment setting.

Reply:
Fixed.